src/content/docs/mobile-monitoring/new-relic-mobile/get-started/introduction-mobile-monitoring.mdx

```diff
@@ -60,36 +60,74 @@ import mobileCrashAnalysis from 'images/mobile_screenshot-full_crash-analysis.we
 
 import mobileNextSteps from 'images/mobile_diagram_next-steps.webp'
 
-New Relic's mobile monitoring capabilities help you gain deeper visibility into how to analyze your [Android](/docs/mobile-monitoring/new-relic-mobile-android/get-started/introduction-new-relic-mobile-android) and [iOS](/docs/mobile-monitoring/new-relic-mobile-ios/get-started/introduction-new-relic-mobile-ios) application performance and troubleshoot crashes. You can also examine HTTP and other network performance for unexpected lag, which will in turn help you collaborate more efficiently with your backend teams.
-
-You'll be able to configure mobile monitoring to suit your environment after you create a New Relic account (it's free, forever!) and [install mobile monitoring](#new_users) for your Android, iOS, or Cordova environment.
-
-<InlineSignup/>
-
-## How we use mobile monitoring [#features]
-
-At New Relic, we have our own mobile developers who create iOS and Android apps so our customers can use New Relic from their mobile devices. During a feature deploy, we encountered a small bug that caused an edge case crash with our Android app. This happened after we had created, tested, and deployed the new feature.
-
-Because we work at New Relic, we also use our monitoring tools to analyze and troubleshoot our mobile apps. To give you an idea of how to explore the power of mobile monitoring, here's an example of how our own mobile engineers used it to track this small bug.
-
-<img
-  title="Crash analysis workflow with mobile monitoring"
-  alt="Crash analysis workflow with mobile monitoring"
-  src={mobileCrashAnalysisWorkflow}
-/>
-
-<figcaption>
-  A workflow to identify and resolve mobile app crashes using New Relic.
-</figcaption>
+New Relic's mobile monitoring capabilities provide deeper visibility into the performance and crash troubleshooting of your [Android](/docs/mobile-monitoring/new-relic-mobile-android/get-started/introduction-new-relic-mobile-android), [iOS](/docs/mobile-monitoring/new-relic-mobile-ios/get-started/introduction-new-relic-mobile-ios), and hybrid applications. You can use mobile monitoring to improve your app's user experience or examine HTTP and network performance to collaborate more effectively with your backend teams.
+
+You'll be able to configure mobile monitoring to suit your environment after you create a New Relic account (it's free, forever!) and install mobile monitoring for your Android, iOS, or hybrid environment. 
+
+Choose your platform-specific page to get started:
+
+## Create an app and install your agent: [#install]
+
```

Comment:
```suggestion
## Create an app and install your agent: [#install]

Choose your platform-specific page to get started:
```

Reply:
Generally we don't use colons for headers, and instead use them with preface sentences before instructions.

Reply:
Since these links aren't directly to the UI (where the user would create an app and install the agent), consider renaming this to a general "Get started with mobile monitoring" or something.